src/AST/ASTIntegerLiteral.ts

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) ZALORA
+//
+// This source code is licensed under the MIT license found in the
+// LICENSE file in the root directory of this source tree.
+
+import { IToken as Token } from '../tokens';
+import ASTExpression from './ASTExpression';
+
+class ASTIntegerLiteral implements ASTExpression {
+  token: Token;
+  value: number;
+
+  constructor(token: Token, value: number) {
+    this.token = token;
+    this.value = value;
+  }
+
+  toNumber(): number {
```

Comment:
This needs to be toString always, in all nodes. The whole entire AST needs to be representable as a string.

Reply:
ahhh ok let me update it.